pages/linux/raw.md

```diff
@@ -0,0 +1,8 @@
+# raw
+
+> Bind a raw character device.
+> More information: <https://manned.org/raw.8>.
+
+- Bind a raw character device to a block device:
+
+`raw /dev/raw/raw{{1}} /dev/{{block_device`
```

Comment:
```suggestion
`raw /dev/raw/raw{{1}} {{/dev/block_device}}`
```

Reply:
Not to sure about this, other than the mistake made.

Reply:
I don't think there are block devices outside of `/dev/`.

Reply:
I'm pretty sure that you can have block devices in other places. Even if not, you can symlink.

Reply:
Thats almost like saying I can store all of my files in /boot

I don't think its a good idea to encourage it at all.

Reply:
But when we don't put `/dev/` in token syntax we're indicating that it's required by the tool.

Reply:
> Thats almost like saying I can store all of my files in /boot

Nothing wrong with storing all your documents in `/boot`!

> But when we don't put /dev/ in token syntax we're indicating that it's required by the tool.

Absolutely. In this instance `{{/dev/block_device}}` is most appropriate as this is a specific example (i.e. in place of something like `{{block_device}}`.